client/app/bundles/course/assessment/submission/pages/SubmissionsIndex/SubmissionsTableRow.jsx

```diff
@@ -245,7 +245,7 @@ const SubmissionsTableRow = (props) => {
         <TableCell style={tableCenterCellStyle}>
           <SubmissionWorkflowState
             className={
-              submission.workflowState !== workflowStates.Graded && 'w-36'
+              submission.workflowState !== workflowStates.Graded && 'w-[100px]'
```

Comment:
Instead of hardcoding a magic value, why not we just go with 'w-fit' for everything? Don't distinguish between graded and non-graded states.

Also add `hover:underline` className for the clickable Chips.

Also update the statistics page chips, the chips currently stretch really wide as the Chip defaults to w-full.